frontend/src/locales/en/translation.json

```diff
@@ -831,6 +830,7 @@
   "HighToLow": "high to low",
   "AZ": "A-Z",
   "ZA": "Z-A",
+  "DownloadFileSuccess": "'{{fileName}}' has been downloaded.",
```

Comment:
There are multiple translation keys without spanish and/or portuguese translations